scss/components/_breadcrumbs.scss

```diff
@@ -34,13 +34,21 @@ $breadcrumbs-item-margin: 0.75rem !default;
 /// @type Boolean
 $breadcrumbs-item-uppercase: true !default;
 
-/// If `true`, adds a slash between breadcrumb links.
+/// If `true`, adds a seperator between breadcrumb links.
 /// @type Boolean
-$breadcrumbs-item-slash: true !default;
+$breadcrumbs-item-separator: true !default;
```

Comment:
Can we do a check for if `$breadcrums-item-slash` is defined and use that if so? That would enable backwards compat with older settings files